cypress/e2e/routing.cy.ts

```diff
@@ -0,0 +1,9 @@
+/* eslint-disable no-undef */
+describe("Routing", () => {
+  it("Navigates to other pages", () => {
+    cy.visit("http://localhost:3000");
+    cy.viewport("iphone-6");
+    cy.get("[data-cy=hamburger-icon]").click();
+    cy.get("[data-cy=team-members-link]").click();
```

Comment:
A proper test needs an assertion, you're missing one here. If your goal is to test navigating to other pages, how do you actually check if this is working? All you're doing now is clicking buttons, not checking the result. See: https://docs.cypress.io/guides/end-to-end-testing/writing-your-first-end-to-end-test#Write-a-real-test